gitpractice/Angular/Tasks/myFirst-app/src/app/app.component.html

```diff
@@ -0,0 +1,19 @@
+<app-user></app-user>
+<div appAttributeDirective>
+    hi
+</div>
+<div *ngIf=" ngDestroyDemoVariable" (click)="ngDestroyDemoVariable = !ngDestroyDemoVariable">
+    <h3>ngOnDestory gets called after this element click and the component gets destroyed</h3>
```

Comment:
don't hardcore any values or plain text in html pages
maintain a constant file and fetch them from their.

Check all your html files and maintain a constant file where you have them all.
import them in their respective component.ts files into a variable and use that variable in html.